components/embl-logo/embl-logo.config.yml

```diff
@@ -7,10 +7,10 @@ status: live
 #
 # Per-variant options
 variants:
+  - name: default
   - name: responsive
```

Comment:
Should we also name the variant `extreme`?

Reply:
That comment was meant for line 11.

Reply:
Ah, yes. Will update.